src/main/java/com/github/mkopylec/sessioncouchbase/persistent/CouchbaseSession.java

```diff
@@ -60,12 +60,12 @@ public static String globalAttributeName(String attributeName) {
 
     @Override
     public long getCreationTime() {
-        return round((double) globalAttributes.get(CREATION_TIME_ATTRIBUTE));
+        return round(new Double(globalAttributes.get(CREATION_TIME_ATTRIBUTE).toString()));
```

Comment:
Hi there, sure. Thanks for reviewing this pull request. First time I try this and wasn't sure about ettiquete or even if this changes would be interesting to someone. When I try to add HttpSession session to a controller I get this when GETing the page:
 : Cleared thread-bound request context: com.github.mkopylec.sessioncouchbase.persistent.RequestWrapper@1ea797fd
2016-09-12 19:09:17.910 ERROR 30557 --- [0.0-9000-exec-2] o.a.c.c.C.[.[.[/].[dispatcherServlet]    : Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception

java.lang.ClassCastException: java.lang.Long cannot be cast to java.lang.Double
       	at com.github.mkopylec.sessioncouchbase.persistent.CouchbaseSession.getLastAccessedTime(CouchbaseSession.java:68) ~[session-couchbase-spring-boot-starter-1.2.1.jar:na]
       	at com.github.mkopylec.sessioncouchbase.persistent.CouchbaseSession.isExpired(CouchbaseSession.java:87) ~[session-couchbase-spring-boot-starter-1.2.1.jar:na]
       	at com.github.mkopylec.sessioncouchbase.persistent.CouchbaseSessionRepository.getSession(CouchbaseSessionRepository.java:101)